bcda/api/requests.go

```diff
@@ -514,17 +515,55 @@ func (h *Handler) bulkRequest(w http.ResponseWriter, r *http.Request, reqType se
 	}
 }
 
-func (h *Handler) validateRequest(resourceTypes []string) error {
+func (h *Handler) getResourceTypes(parameters middleware.RequestParameters, cmsID string) []string {
+	resourceTypes := parameters.ResourceTypes
```

Comment:
see below re: resourceType vs. ResourceType confusion

Reply:
@katyasoup yeah I agree, the `resource type` seems to have a couple meanings and can be confusing depending on the context.  I was thinking about changing up the labels here to the following, what do you think?

resource type = FHIR resource type (eg Patient, EOB, Claim, Coverage, etc)
data type = upstream source data (eg adjudicated vs pre-adjudicated)

Reply:
That sounds great! Then we'd have a map of `[resourceType]dataType` which I think is much easier to follow